lib/ClangImporter/ImportMacro.cpp

```diff
@@ -334,6 +345,29 @@ static ValueDecl *importMacro(ClangImporter::Implementation &impl,
         tokenI[3].is(clang::tok::r_paren)) {
       return importStringLiteral(impl, DC, macro, name, tokenI[2],
                                  MappedStringLiteralKind::CFString, ClangN);
+    // Check for literals with a cast, e.g. "(cpu_type_t) 1"
+    } else if (tokenI[0].is(clang::tok::l_paren) &&
+               tokenI[1].is(clang::tok::identifier) &&
```

Comment:
I'm not a clang expert, but are builtin type keywords like `int`, `float`, `char`, etc. lexed as `tok::identifier`? I'd expect them to come in as keyword tokens. You might also want to handle `(unsigned int)`, `(char signed)`, `(long long)`, etc.